src/icp_canister_backend/tests/deposit_tests.rs

```diff
@@ -300,46 +204,22 @@ fn test_successful_withdrawal() {
         owner: user,
         subaccount: None,
     };
-    let initial_balance_result = pic
-        .query_call(
-            ledger_id,
-            user,
-            "icrc1_balance_of",
-            encode_args((user_account.clone(),)).unwrap(),
-        )
-        .expect("Failed to check user initial balance");
-    let initial_balance: Nat = decode_one(&initial_balance_result).unwrap();
+    let initial_balance = client.connect(user).icrc1_balance_of(user_account.clone());
 
     // Create deposit and advance time to simulate finished episode
-    let current_episode = get_stakable_episode(&pic, canister_id, 0);
-    create_deposit(
-        &pic,
-        canister_id,
-        ledger_id,
-        user,
-        deposit_amount.clone(),
-        current_episode,
-    );
-    let current_episode_time_to_end = get_episode_time_to_end(&pic, current_episode);
-    advance_time(&pic, current_episode_time_to_end);
+    let current_episode = client.get_stakable_episode( 0);
```

Comment:
( 0) means you didn't launch a formatter for the file.
Make sure all the files are formatted.